_docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -116,7 +116,8 @@ sleep     1         1         1            1           2h        sleep,istio-pro
 
 ## Automatic sidecar injection
 
-See [validatingadmissionwebhook-alpha-in-18-beta-in-19](https://kubernetes.io/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-18-beta-in-19) for overview of webhook admission controller.
+See the [validatingadmissionwebhook-alpha-in-18-beta-in-19](https://kubernetes.io/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-18-beta-in-19) section which lists the webhook admission controllers required for automatic sidecar injection.
```

Comment:
Can we use a more human-readable link name here?

Reply:
See the [validatingadmissionwebhook](https://kubernetes.io/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-18-beta-in-19) documentation for the list of admission controllers required to enable mutating webhooks.

Reply:
Used "ValidatingAdmissionWebhook" since that is the kubernetes.io section name.